tools/compress.cpp

```diff
@@ -9,22 +9,28 @@
 #include <spdlog/sinks/stdout_color_sinks.h>
 #include <spdlog/spdlog.h>
 
+#include "CLI/CLI.hpp"
 #include "app.hpp"
-#include "create_freq_index.hpp"
+#include "compress.hpp"
 #include "index_types.hpp"
 #include "util/index_build_utils.hpp"
 #include "util/util.hpp"
 #include "wand_data.hpp"
 #include "wand_data_raw.hpp"
 
-#include "CLI/CLI.hpp"
-
 int main(int argc, char** argv)
```

Comment:
Could we possible name this file/tool `compress_inverted_index` or maybe `compress_index` perhaps? I feel like `compress` is not quite verbose enough... Though maybe it doesn't matter. @elshize, any preference?

Reply:
Either works for me.

Reply:
> Either works for me.

You mean any of the three options? :-)

Reply:
I'd lean towards adding at least "index". Though I don't think it matters much. What I'd like to suggest is to start using hyphens instead of underscores in the command names, such as `compress-index`. That's usually how it goes, and it would be nice to follow the standard way.

Reply:
Agreed with `compress_inverted_index`